frontend/main.js

```diff
@@ -178,11 +191,17 @@ async function startApp () {
           syncs: [],
           // TODO/REVIEW page can load with already loggedin. -> this.$store.state.loggedIn ? 'yes' : 'no'
           finishedLogin: 'no',
+          debouncedSyncBanner: null,
           isCorrupted: false // TODO #761
         }
       }
     },
     mounted () {
+      if (sbp('okTurtles.data/get', 'BANNER')) {
+        // mounted gets called each time we logout, so return immediately if we've already initialized
```

Comment:
Really? Wow

Reply:
I think this might be a relic of me going nuts with debugging as there were a **LOT** of asynchronous issues that had to be worked out in this PR to get the Cypress tests passing.

I just fixed all the issues (at least the ones myself and the Cypress tests were able to find), so I'll be removing this as they're passing with it removed, and this really should never happen as you rightfully noticed.